docs/plugins/regrass.rst

```diff
@@ -5,15 +5,56 @@ regrass
     :summary: Regrow surface grass and cavern moss.
     :tags: adventure fort armok animals map
 
-This command can refresh the grass (or subterranean moss) growing on your map.
-It will only regrow the grass and moss on tiles that had some to begin with.
+This command can refresh the grass (and subterranean moss) growing on your map.
+Operates on floors, stairs, and ramps. Also works underneath shrubs, saplings,
+and tree trunks. Ignores furrowed soil, beaches, and tiles under buildings.
 
 Usage
 -----
 
 ::
 
-    regrass [max]
+    regrass [options]
 
-Specify the ``max`` keyword to pack more grass/moss onto a tile than what the
-game normally allows. This will give your grazers extra chewing time.
+Regrasses the entire map by default.
+
+``debugfilter set Debug regrass log`` can be used to figure out why regrass
+is failing on a tile. (Avoid regrassing the entire map with this enabled, as it
+will make the game unresponsive and flood the console for several minutes!)
+Disable with ``debugfilter set Info regrass log``.
+
+Options
+-------
+
+``h``, ``help``
+    Display this help screen.
+``m``, ``max``
+    Maxes out every grass type in the tile, giving extra grazing time.
+    Not normal DF behavior. Tile will appear to be the first type of grass
+    present in the map block until that is depleted, moving on to the next type.
+    When this option isn't used, a grass type will be selected randomly instead.
+``n``, ``new``
+    Adds biome-compatible grass types that were not originally present in the
+    map block. Allows regrass to work in blocks that never had any grass to
+    begin with. Will still fail in incompatible biomes.
```

Comment:
can there be a `-f`, `--force` option to add grass despite being in an incompatible biome? some players have requested that this tool be able to add grass to, for example, mountain biomes

Reply:
What type of grass should it use? User-specified?

Reply:
optionally user specified would be a nice to have, but at least random. I'd think the common case here would be "I want some grass because my grazers are starving"